107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -129,7 +129,7 @@ public void jsr107LoaderGetsRegistered() {
     merger.mergeConfigurations("cache", configuration);
 
     assertThat(factory.called, is(true));
-    verify(cacheLoaderWriterFactory).registerJsr107Loader(eq("cache"), Matchers.<CacheLoaderWriter<Object, Object>>anyObject());
+    verify(cacheLoaderWriterFactory).registerJsr107Loader(eq("cache"), ArgumentMatchers.<CacheLoaderWriter<Object, Object>>any());
```

Comment:
Since this is a verification we should probably use `any(CacheLoaderWriter.class)` to ensure the loader writer is not null.